tests-examples/demo-todo-app.spec.ts

```diff
@@ -0,0 +1,437 @@
+import { test, expect, type Page } from '@playwright/test';
```

Comment:
Do we need this?

Reply:
Nope! Considered keeping it in for reference, but if we'd prefer to keep the codebase clean then I'm up for taking it out.